### ISSUE TITLE
HTML <mark> tag disappearing

### ISSUE THREAD

**Issue by jolrael-dev** [nhn/tui.editor#711]
## Version
1.3.0

## Development Environment
Windows and Mac: Chrome, Firefox, and edge.

## Current Behavior
When a user puts a <mark> tag in the markdown tab it shows the text highlight correctly on the preview, but when the tab is switched to the wysiwyg the highlight disappears. The tag is no longer on the text when you go back to the markdown tab.

## Expected Behavior
The mark tag stays in place like html tags or is a special type like italic and bold.

**Comment by jolrael-dev** [comment]
The mark tag is an HTML5 tag, but it's confusing my users that they can see it and then it goes away
https://www.w3schools.com/tags/tag_mark.asp
![image](https://user-images.githubusercontent.com/58951019/70943020-7d7d7f00-2015-11ea-9e9c-ee2601590c0a.png)

**Comment by Seunghyum** [comment]
I tried like you tried. However It is no problem. Can I see your code by codepen or jsbin etc?
I tried at Chrome and Safari.

![스크린샷 2019-12-18 오후 1 21 42](https://user-images.githubusercontent.com/13027315/71055559-741d1f80-2199-11ea-952c-090a8ba4efd8.png)
![스크린샷 2019-12-18 오후 1 22 00](https://user-images.githubusercontent.com/13027315/71055560-741d1f80-2199-11ea-8ce7-2e532b28995c.png)

**Comment by jolrael-dev** [comment]
I'm able to reproduce it here https://nhn.github.io/tui.editor/latest/tutorial-example01-editor-basic
On Markdown tab:
![image](https://user-images.githubusercontent.com/58951019/71123354-104e2300-21a8-11ea-8366-cfa5cc0eaa9d.png)
After changing to WYSIWYG tab:
![image](https://user-images.githubusercontent.com/58951019/71123481-6327da80-21a8-11ea-8d28-1feac0db8a88.png)

**Comment by jongfeel** [comment]
@jolrael-dev 
Yes, you're right. Mark tag is now showing in WYSIWYG tab. also removed mark tag when back to Markdown tab

**Comment by seonim-ryu** [comment]
This is a question related to the issue #705. Please check that issue first. Currently, WYSIWYG is being removed except for the tag below. And in WYSIWYG, the tags below and other tags are removed. The `mark` tag is the case here.

> https://github.com/seonim-ryu/Squire/blob/fd40b4e3020845825701e9689f190bab3f4775d4/build/squire-raw.js#L2047

**Comment by jolrael-dev** [comment]
The html5 tag mark does not carry over to the wysiwyg tab, not showing the literal text or displaying the highlight. To contrast the tag \<sub> will still show the text as a subscript on the wysiwyg tab, and the html5 time tag stays when you return to the markdown tab.
1 markdown tab:
![image](https://user-images.githubusercontent.com/58951019/71212328-b700f500-2276-11ea-8658-f7709f0cf621.png)
2 wysiwyg tab:
![image](https://user-images.githubusercontent.com/58951019/71212367-cbdd8880-2276-11ea-87d9-b6775ac0ea85.png)
3 back to markdown tab:
![image](https://user-images.githubusercontent.com/58951019/71212428-e7489380-2276-11ea-8737-7b3c88813a28.png)

As you can see the fake tag and the mark tag was removed, but time and sub were still there. Mark is an html5 tag and should stay.

**Comment by seonim-ryu** [comment]
@jolrael-dev I got it.. It's because of the tag handling in Squire. However, what I said before was the processing of block elements. `mark` is an inline element, which is excluded from the logic of processing inline elements in Squire.

> https://github.com/seonim-ryu/Squire/blob/fd40b4e3020845825701e9689f190bab3f4775d4/build/squire-raw.js#L182

Simply adding this tag to your code won't solve the problem. I think this issue can be solved by specifying the specification of which tags should be allowed or not. I'll let you know when it's decided.
